apps/main-landing/src/app/creators/donate/components/select/select-option.component.tsx

```diff
@@ -17,7 +17,9 @@ export const SelectOption = forwardRef(
           className,
         )}
       >
-        {option.prefix}
+        <div className='relative pr-2 mr-1 after:absolute after:right-0 after:-top-[6px] after:bg-gray-200 after:w-px after:h-[calc(2.625rem_-_6px)]'>
```

Comment:
please change the `after:-top-[6px]` to `after:top-[-6px]`

Reply:
I changed it to after:-top-1.5